issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraIssue.java

```diff
@@ -237,18 +238,24 @@ private Map<String, String> availableTransitions() {
                                                     v -> v.get("id").asString()));
     }
 
-    private void performTransition(String id) {
+    private void performTransition(String state) {
+        var id = availableTransitions.get(state);
         var query = JSON.object()
-                        .put("transition", JSON.object()
-                                               .put("id", id));
+                .put("transition", JSON.object()
+                        .put("id", id));
+        if (state.equals("RESOLVED")) {
```

Comment:
This method is called with `"Resolved"` not `"RESOLVED"`. To avoid relying on string comparisons here, could we make all the calls to this method use constants, or perhaps even an internal enum?

Reply:
Sure